Code/UnitTests/RendererTest/CMakeLists.txt

```diff
@@ -12,13 +13,31 @@ target_link_libraries(${PROJECT_NAME}
   PUBLIC
   TestFramework
   RendererCore
-  RendererDX11
 )
 
-ez_link_target_dx11(${PROJECT_NAME})
+if (EZ_BUILD_EXPERIMENTAL_VULKAN)
+  target_link_libraries(${PROJECT_NAME}
+    PUBLIC
+    RendererVulkan
+  )
+
+  add_dependencies(${PROJECT_NAME}
+    ShaderCompilerDXC
+  )    
+endif()
+
+if (EZ_CMAKE_PLATFORM_WINDOWS)
+  target_link_libraries(${PROJECT_NAME}
+    PUBLIC
+    RendererDX11
+  )
+  ez_link_target_dx11(${PROJECT_NAME})
+  
+  add_dependencies(${PROJECT_NAME}
+    ShaderCompilerHLSL
+  )
+endif()
```

Comment:
Do we need a helper function for all this?

Reply:
yeah, might be worth doing going forward